<launch>
  <!-- load params -->
  <rosparam
    command="load"
    ns="control"
    file="$(find pandora_sensor_orientation_controller)
    /config/sensor_orientation_controller.yaml"/>

  <node
    name="kinect_orientation_controller"
    pkg="pandora_sensor_orientation_controller"
    type="sensor_orientation_controller"
    respawn="false"
    output="screen"
    ns="control"
    args="move_kinect_action"/>

  <!-- <node                                -->
  <!--   name="head_orientation_controller"    -->
  <!--   pkg="pandora_sensor_orientation_controller" -->
  <!--   type="sensor_orientation_controller"  -->
  <!--   respawn="false"                    -->
  <!--   output="screen"                    -->
  <!--   ns="control"                       -->
  <!--   args="move_head_action"/>          -->

</launch>
